gradle/verification-metadata.xml

```diff
@@ -281,36 +281,76 @@
             <sha256 value="e8f1e75d1355e2223d682224c45563b74da67f8638484960e04d7b2313919264" origin="Generated by Gradle"/>
          </artifact>
       </component>
+      <component group="com.github.haveno-dex.netlayer" name="tor" version="72a4fae5cf">
```

Comment:
Mind removing the old dependencies as well?

Reply:
Dependencies are removed